core/controllers/editor_test.py

```diff
@@ -556,8 +556,8 @@ def test_exploration_download_handler_for_default_exploration(self):
 
         # Check download to JSON.
         exploration.update_objective('Test JSON download')
-        exp_services._save_exploration(  # pylint: disable=protected-access
-            owner_id, exploration, '', [])
+        exp_services.update_exploration(
+            owner_id, exploration.id, None, 'Test commit')
```

Comment:
Try and use the correct "expected" changelist, here and elsewhere.

Reply:
Sorry, I didn't get it. What do you mean by "correct expected changelist" here?

Reply:
Look through the codebase to see what gets passed as the changelist arg during normal operation. Then, use that arg here as well.